docs/changes.rst

```diff
@@ -22,6 +22,13 @@ You should generally assume that an API is internal unless you have specific
 information to the contrary.
 
 -------------------
+3.11.7 - 2017-06-27
```

Comment:
Adding a new strategy means that we should bump the minor version, which would make this 3.12.0 - don't forget to do version.py too.